book/02-git-basics/sections/getting-a-repository.asc

```diff
@@ -7,7 +7,25 @@ The second clones an existing Git repository from another server.
 
 ==== Initializing a Repository in an Existing Directory
 
-If you're starting to track an existing project in Git, you need to go to the project's directory and type
+If you're starting to track an existing project in Git, you need to go to the project's directory
```

Comment:
I totally agree with you. Your way of explanation is much better than what I did (just to add the examples). What do you prefer:
1. Me to change the pull request with additional commit?
2. You to create an additional commit after you merge the PR?
3. Anything else?